core/domain/stats_services.py

```diff
@@ -1124,6 +1124,87 @@ def get_learner_answer_details(entity_type, state_reference):
     return None
 
 
+def get_learner_answer_info_for_exploration(exp_id):
+    """Returns the learner answer info in dict format.
+
+    Args:
+        exp_id: str. The ID of the exploration.
+
+    Returns:
+        list. Consists of dicts
```

Comment:
list(dict).

But why are you returning dicts? You should be returning domain objects here. Dicts should only be used at the application boundary (for data transmission across URLs, or to the GAE datastore storage layer).

Reply:
Done

Reply:
Made services to return domain object only